supervisor/backups/manager.py

```diff
@@ -506,8 +506,8 @@ async def do_restore_full(
 
         if backup.supervisor_version > self.sys_supervisor.version:
             raise BackupInvalidError(
-                f"Backup was made on supervisor version {backup.supervisor_version}, "
-                f"can't restore on {self.sys_supervisor.version}. Must update supervisor first.",
+                f"Backup was created with newer Supervisor than the one currently installed ({self.sys_supervisor.version})."
+                f"Update Supervisor to at least {backup.supervisor_version}, then try restore the backup again.",
```

Comment:
The only odd part about this is you can't update supervisor to a specific version. The only thing you can do is update supervisor to the latest version, whatever it is. That was why I originally phrased it just saying "Must update supervisor" since that's the only possible action. But I guess I'm ok with this too.